README-nethermind.md

```diff
@@ -0,0 +1,58 @@
+# EWC validator node install scripts
```

Comment:
I wrote it on slack:

> An information which is missed for me is a difference between validators.
> When user go to repo he will see 2 validator types.
> I think it's confusing when user doesn't have additional explanation.
> Even I don't know which one is better and why :P

> Also it's worth to mention about nethermind in README.md because github shows this file first in a preview.